data/projects/implement_aspects.md

```diff
@@ -7,9 +15,26 @@ The aspects concept exists as
 already possible to define aspects and use them in bears - however this has no
 effect because coala doesn't understand what to do with them yet.
 
```

Comment:
The idea on how aspect work is already stated now. I need to do my homework first for the implementation explanation 😅